Tests/eZ/Publish/FieldType/TweetTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
```

Comment:
Don't hesitate to use several commits for each tag, if needed. (If you work on the commits history and order, I put my foot in the door to add my wishes). I am thinking about something like this:

```
$> git log

commit <hash>
Creation of the class Type.php

commit <hash>
Adding this methods

commit <hash>
Tests for the class Type (optional in the Tutorial)
```


And the third commit is tagged as the stepN (_step4_implement_the_tweet_type_class_ ie) of the Tutorial. It put the question of the identification of the commits messages to have the tags on the table again.